hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -20,7 +20,7 @@
 <jboss-deployment-structure>
   <deployment>
     <dependencies>
-      <module name="org.hawkular.bus" />
+      <module name="deployment.hawkular-alerts-rest.war" />
```

Comment:
Maybe what we can do is merge this PR into master, rebase the m+a PR on master, and then update the dependencies to work with the m+a PR.  In that way, if we merge that PR it will work and not break with the m+a distribution.  But then it will not work for standalone alerting (which is the secondary use case) and we will have to figure that out.